coa/pkg/apis/v1alpha2/bindings/http/metrics/metrics.go

```diff
@@ -15,14 +15,14 @@ import (
 
 // Metrics is a metrics tracker for an api operation.
 type Metrics struct {
-	apiOperationLatency observability.Histogram
+	apiOperationLatency observability.Gauge
```

Comment:
Same as previous comment on using Histogram - is there a reason to switch to gauge?

Reply:
This is due to some external collectors such as Geneva doesn't well support otel-histogram. If we use histogram to emit the data, the data in Geneva is wrong. We have reported this issue to ME team. But it still needs several months to consume the release. Before that, let's use gauge at first.